BinaryEncoding.md

```diff
@@ -65,20 +65,36 @@ Yes:
 
 ## Global structure
 
-* A module contains:
-  * a header followed by
-  * a table (sorted by offset) containing, for each section, its type and offset (within the module), followed by
-  * a sequence of sections.
+* A module contains (in this order):
+  * A header
```

Comment:
Yeah that sounds good. I like what you're adding overall, so I'll step back, take this improvement, and we can iterate later :-)